static/js/components/ExpandedLearningResourceDisplay.js

```diff
@@ -246,6 +251,28 @@ const ExpandedLearningResourceDisplay = (props: Props) => {
           ))}
         </div>
       ) : null}
+      {similarItems ? (
+        <div className="expanded-learning-resource-list">
+          <div className="list-header">
+            Similar Learning Resources
+            <i
+              className={`material-icons float-right ${similarIcon}`}
+              onClick={() => setShowSimilar(!showSimilar)}
+            >
+              {similarIcon}
+            </i>
+          </div>
+          {showSimilar
+            ? similarItems.map((resource, i) => (
+              <LearningResourceRow
+                key={i}
+                object={searchResultToLearningResource(resource)}
+                searchResultLayout={SEARCH_LIST_UI}
+              />
+            ))
+            : null}
+        </div>
+      ) : null}
```

Comment:
two questions:

- I seem to remember some discussion that the list of items, in the case of a program, userlist, or learningpath, should collapse similarly to the similar items list. would it make sense to add this behavior in while we're in here?
- also since we have some duplicate code here between the items list and the similar items list, it might be helpful to create a separate component in this same file which takes like a title and list of items and does the rest. what do you think?

Reply:
Sure, I'll make those changes

Reply:
Changes made.  @abdulkdawson here's a screenshot of both userlist/program items and similar resources, in their default collapsed state - I put 5px of padding between them, does that look okay?

<img width="609" alt="Screen Shot 2019-12-13 at 2 05 30 PM" src="https://user-images.githubusercontent.com/187676/70831406-7f023980-1dc0-11ea-960c-0119852d78cc.png">

Reply:
@aliceriot ready for review again